Dalamud/Storage/ReliableFileStorage.cs

```diff
@@ -22,12 +22,14 @@ namespace Dalamud.Storage;
 /// This is not an early-loaded service, as it is needed before they are initialized.
 /// </remarks>
 [ServiceManager.ProvidedService]
-public class ReliableFileStorage : IServiceType, IDisposable
+public class ReliableFileStorage : IPublicDisposableService
```

Comment:
I actually think that this is public by mistake, sorry... It should probably be internal. Maybe it's good to keep IPublicDisposableService around, since we might need it in the future - though realistically, we should probably try to keep as much complexity as possible out of Service<T> until we need it.

Reply:
Marked it obsolete, and added to TODO that we should make it internal on api 10.